erpnext/stock/report/stock_ledger/stock_ledger.py

```diff
@@ -20,18 +20,16 @@
 def execute(filters=None):
 	is_reposting_item_valuation_in_progress()
 	if filters.get("include_uom") and filters.get("type") != "Default UOM":
-		frappe.throw(str("Cannot select filters <b> Include UOM </b> and <b>Select UOM</b> together!"))
+		frappe.throw(
+			_(
+				f"Cannot select filters {frappe.bold ('Include UOM')} and {frappe.bold('Select UOM')} together!"
+			)
+		)
```

Comment:
I think the function calls should be added as separately translated variables, something like this:

```
include_str = _('Include UOM').bold
select_str =  _('Select UOM').bold
frappe.throw(
   _("Cannot select filters {0} and {1} together!".format(include_str, select_str)
)
```

Reply:
@Dilshanavs ^